### ISSUE TITLE
Add documentation on how to source and run/use esmvaltool from HPC central installations

### ISSUE THREAD

**Issue by valeriupredoi** [ESMValGroup/ESMValCore#989]
Nice to have HPC cluster-wide installations users can just use and run without the headache of installing it themselves; we've done that on CEDA-JASMIN by installing a module that users can just load by `module load esmvaltool`. Once this is done elsewhere we should update the documentation.

- JASMIN done in https://github.com/ESMValGroup/ESMValCore/pull/841
- DKRZ: done in https://github.com/ESMValGroup/ESMValCore/pull/1155
- ETHZ: @bascrezee maybe?
- any other places?

Incidentally, I have written some instructions on how to create a working module in this [comment](https://github.com/ESMValGroup/ESMValTool/issues/1888#issuecomment-738018548) if people would need to build their own centrally-deployed modules in a shared space like we did with @agstephens at JASMIN

**Comment by Peter9192** [comment]
I've been setting up something similar for the JupyterLab instance at DRKZ, might be nice to combine these things.

**Comment by valeriupredoi** [comment]
yip, good idea!

**Comment by remi-kazeroni** [comment]
The command `module load esmvaltool` is now working at DKRZ as well 🥳 Feel free to try it out and let me know if it works fine @ESMValGroup/esmvaltool-coreteam 

@valeriupredoi I see that we have the instructions for Jasmin on the ESMValCore [documentation](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/quickstart/install.html#pre-installed-versions-on-hpc-clusters). Should this be also in the ESMValTool [documentation](https://docs.esmvaltool.org/en/latest/quickstart/installation.html#installation)? Just thinking where to add the DKRZ info.

**Comment by Peter9192** [comment]
> 
> 
> The command `module load esmvaltool` is now working at DKRZ as well 🥳 Feel free to try it out and let me know if it works fine @ESMValGroup/esmvaltool-coreteam
> 
> @valeriupredoi I see that we have the instructions for Jasmin on the ESMValCore [documentation](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/quickstart/install.html#pre-installed-versions-on-hpc-clusters). Should this be also in the ESMValTool [documentation](https://docs.esmvaltool.org/en/latest/quickstart/installation.html#installation)? Just thinking where to add the DKRZ info.

**Comment by Peter9192** [comment]
Sorry wrong button!

**Comment by valeriupredoi** [comment]
> The command `module load esmvaltool` is now working at DKRZ as well partying_face Feel free to try it out and let me know if it works fine @ESMValGroup/esmvaltool-coreteam
> 
> @valeriupredoi I see that we have the instructions for Jasmin on the ESMValCore [documentation](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/quickstart/install.html#pre-installed-versions-on-hpc-clusters). Should this be also in the ESMValTool [documentation](https://docs.esmvaltool.org/en/latest/quickstart/installation.html#installation)? Just thinking where to add the DKRZ info.

awesome :partying_face: Yeah that's a good idea, dude!

**Comment by valeriupredoi** [comment]
just passing by this in passing (terrible pun, I know :rofl: ) but do we still want to work on this? ie @Peter9192 did you have a chance to add your JupyterLab instructions or do we have anything moving at ETHZ?

**Comment by Peter9192** [comment]
I was thinking to update the JupyterLab instructions somewhen after the new release. Can we keep it open for a bit longer?

**Comment by ruthlorenz** [comment]
I do not think there has been anything moving at ETHZ, Bas is not here anymore :-(
We have talked about it with our sysadmins but at the time the decision was not to have a central installation (mainly because all the users were developers, so it would not help much). But maybe @lukasbrunner wants to look into it again?
